nall/terminal.hpp

```diff
@@ -4,59 +4,56 @@
 
 namespace nall::terminal {
 
-inline auto escapable() -> bool {
-  #if defined(PLATFORM_WINDOWS)
-  //todo: colors are supported by Windows 10+ and with alternate terminals (eg msys)
```

Comment:
This comment shouldn't be lost, but instead reworded into a note instead of a todo: explaining why colours are disabled by default on Windows